board/hx20/battery.c

```diff
@@ -363,8 +363,9 @@ static enum ec_status cmd_charging_limit_control(struct host_cmd_handler_args *a
 		charging_maximum_level = charging_maximum_level | CHG_LIMIT_OVERRIDE;
 
 	if (p->modes & CHG_LIMIT_GET_LIMIT) {
-		system_get_bbram(SYSTEM_BBRAM_IDX_CHG_MAX, &charging_maximum_level);
-		r->max_percentage = charging_maximum_level;
+		uint8_t max = 0;
```

Comment:
fwiw I think the code style here is to declare all variables at the beginning of the function

Reply:
Fair. I was iffy on whether I wanted to introduce a temporary anyway, given that `ec_response_chg_limit_control::max_percentage` is itself a `uint8_t`. I switched over to reading directly into it.

This may cause a compat problem if the response from this command ever changes, but the correct way to handle that would be with a new command version anyway . . . :smile:

Thanks!